lib/docs/stories/introduction/usage.stories.mdx

```diff
@@ -1,89 +1,171 @@
 import { Meta } from "@storybook/addon-docs/blocks";
+import Screenshot from "../assets/screenshot2.png";
+import { Tabs, Tab, TabButton } from "../../tabs";
 
-<Meta title="Introduction/Usage" />
+<Meta title="Introduction/Getting Started" />
 
-# Usage
+# Getting Started
 
-Set up and use PubNub Chat Components for React to build your own chat application.
+Create a sample chat app with [Message Input](https://pubnub.github.io/react-chat-components/docs/?path=/docs/components-message-input--default) and [Message List](https://pubnub.github.io/react-chat-components/docs/?path=/docs/components-member-list--default) components.
+
+  <img src={Screenshot} alt="PubNub Chat Components" style={{ margin: "-20px 0 0 -54px" }} />
 
 ## Requirements
 
-- React v16.8+ and ReactDOM v16.8+
-- [PubNub JavaScript SDK v4.29+](https://www.pubnub.com/docs/sdks/javascript/)
-- [PubNub React SDK v2.1.0+](https://www.pubnub.com/docs/chat/react/setup)
+You will need these tools:
 
-## Set up PubNub account
+- [npm](https://docs.npmjs.com/cli/v6/commands/npm-install)
+- [Node.js](https://nodejs.org/en/download/)
+- Code editor (e.g. [Visual Studio Code](https://code.visualstudio.com/download))
 
-1. Sign in or create an account to create an app on the
-   [Admin Portal](https://dashboard.pubnub.com/) and get the keys to use in your application.
+## Steps
 
-2. When you create a new app, the first set of keys is generated automatically, but a single app can
-   have as many keysets as you like. We recommend that you create separate keysets for production
-   and test environments.
+Get your keysets and use them to run an app. 
```

Comment:
Lukasz G. found this sentence confusing when going through getting started, figured we should remove it and start with "Create a PubNub account"